### DISCUSSION TITLE
Format of the sql file

### DISCUSSION THREAD

**Discussion by akhil2217** [rwnx/pynonymizer#156] [Q&A]
Hey I am trying to use the pynonymizer library .Can some one tell me what is the format of the input sql file which is passed to the function?
Also Is it necessary to install the postgres-client or mysql-client to run the pynonymizer is there no workaround I just want to anonymize the data

**Comment by rwnx** [comment]
Hi, the SQL input file is a dump file from mysqldump or pgdump respectively. 

Yes postgres-client or mysql-client are dependencies. You could try the beta docker images at https://hub.docker.com/r/rwnxt/pynonymizer if you dont want to add dependencies. 
If you're just anonymizing an existing database and you dont want to use input or output you can use `--only-step ANONYMIZE_DB`.

**Comment by rwnx** [comment]
Following the discussion on this issue: https://github.com/rwnx/pynonymizer/issues/158

It's likely your use case is not supposed to require an input file at all. As a workaround, if you provide a dummy sql file with --only-step ANONYMIZE_DB, it should not be used

**Reply by akhil2217** [reply]
Hi Rowan Thank you so much for your reply .
I have restrictions installing the mysql or postgres client on my machine so I am not able to generate the sql dump file .I wanted to see an example sql file for reference purpose . So that I could create one for myself and run the library to generate the output.sql.
